annotation_full.wdl

```diff
@@ -263,38 +265,37 @@ task stage {
 }
 
 task make_map_file {
-
+  input{
     String proj_id
-    String  prefix=sub(proj_id, ":", "_")
+    String prefix=sub(proj_id, ":", "_")
     File input_file
     String container
-    String output_file = "${prefix}_map.fasta"
     Int min_seq_length = 150      # default value
     Int unknown_gap_length = 100  # default value
-
+  }
 
   command <<<
-  find_prefix=`grep ${proj_id} ${input_file} | head -1`
+  find_prefix=`grep ~{proj_id} ~{input_file} | head -1`
 
   set -euo pipefail
   if [[ $find_prefix ]]
   then
     echo "false" > run_map.txt
-    ln ${input_file} ${output_file} || ln -s ${input_file} ${output_file}
+    ln ~{input_file} ~{prefix}_map.fasta || ln -s ~{input_file} ~{prefix}_map.fasta
   else
     echo "true" > run_map.txt
```

Comment:
we should always be generating the mapping file so I don't think we need this extra logic about find_prefix